<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-NDH55H9" name="com.TestNgData_Provider.DataFrom_excel" tests="1" failures="0" timestamp="2022-09-28T22:25:13 IST" time="0.000" errors="1">
  <testcase name="Readerr" time="0.000" classname="com.TestNgData_Provider.DataFrom_excel">
    <error type="org.testng.TestNGException" message="
Some DataProvider public static java.lang.String[][] com.TestNgData_Provider.TestDataFromExcel.DataProvider_ALLDATA(int) throws java.io.IOException parameters unresolved:  at 0 type int
">
      <![CDATA[org.testng.TestNGException: 
Some DataProvider public static java.lang.String[][] com.TestNgData_Provider.TestDataFromExcel.DataProvider_ALLDATA(int) throws java.io.IOException parameters unresolved:  at 0 type int

at org.testng.internal.MethodInvocationHelper.getParameters(MethodInvocationHelper.java:220)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:145)
at org.testng.internal.Parameters.handleParameters(Parameters.java:798)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Readerr -->
  <system-out/>
</testsuite> <!-- com.TestNgData_Provider.DataFrom_excel -->
